acrobat/blocks/app-banner/app-banner.js

```diff
@@ -0,0 +1,128 @@
+import { createTag } from "../../scripts/miloUtils.js";
+
+var link = '';
+var mobileOS = getMobileOperatingSystem();
+
+
+export default async function init(el) {
+  //Exit if not mobile/tablet 
+  if (mobileOS == 'unknown') {
+    el.innerHTML = '';
+    return;
+  }
+
+  createTag.then((createTag) => {
+    const bannerPlaceHolder = createTag('div', { class: 'placeholder-banner' }, '');
+    document.body.prepend(bannerPlaceHolder);
+    const children = el.querySelectorAll(':scope > div');
```

Comment:
if you need children of the element you can use `el.children`

Reply:
I need only the divs, I have some srcset and other elements also.

Reply:
Tried it, ignoring srcset anyway.